pkg/sentry/socket/unix/transport/connectionless.go

```diff
@@ -108,7 +109,7 @@ func (e *connectionlessEndpoint) SendMsg(ctx context.Context, data [][]byte, c C
 	}
 
 	opts := UnixSocketOpts{}
-	connected, err := to.UnidirectionalConnect(ctx, opts)
+	connected, err := to.UnidirectionalConnect(ctx, opts, nil)
```

Comment:
@ayushr2 you see that I added a pointer type `*lisafs.KUIDGID` to many function signatures. This is because [ClientFD.Connect()](https://github.com/google/gvisor/blob/de62fb56fce68bf8c4b221a1e0c0b7df3bd69499/pkg/lisafs/client_file.go#L475) is transitively used here, and here in [SendMsg()](https://github.com/google/gvisor/blob/de62fb56fce68bf8c4b221a1e0c0b7df3bd69499/pkg/lisafs/client_file.go#L475) the uid/gid are not so accessible.

As go does not natively provide the `Option[T]` type, I use the pointer type and when `nil` is passed, on [ClientFD.Connect()](https://github.com/google/gvisor/blob/44562d85cf0642fb83ec30f27b450d2fcaa9a722/pkg/lisafs/client_file.go#L470) the old RPC `Connect` is used without credentials.

I am a newbie in go. Let me know if you have a better idea :D